ui/src/data-services/models/deployment.ts

```diff
@@ -9,6 +9,11 @@ export class Deployment {
     this._deployment = deployment
   }
 
+  get canDelete(): boolean {
+    // TODO: Check delete permission instead of update
+    return this._deployment.user_permissions.includes(UserPermission.Update)
```

Comment:
The delete permission is actually available in the API! It shows for superusers only
<img width="221" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/158175/8a57a78f-2e46-406f-82a8-61baea31707b">